sh/tests/remake_test.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
```

Comment:
You can try to do it as we did for example for the "file-struct" exercise for example, where we ask for a .tar file and then compare them.

If you choose this way, you must deal with this:
![image](https://user-images.githubusercontent.com/53977751/212370096-69de2afc-1959-4dc1-bb30-ca44b7b1803a.png)

I purposely did a wrong solution for the student, and run the test, it failed like it should but
after that if you don't remove the created files you cant run the tests again.
so i guess here in the test you need to fix that.

Reply:
I have slightly changed the subject and the tests to avoid these types of errors - it is asked now to check if the directory exists before creating the asked files + the tests clears up the created files after running the `challenge` funciton